messenger.rst

```diff
@@ -839,11 +839,73 @@ Transport Configuration
 -----------------------
 
 Messenger supports a number of different transport types, each with their own
-options.
+options. Options can be passed to the transport via a DSN string or configuration.
+
+.. code-block:: bash
+
+    # .env
+    MESSENGER_TRANSPORT_DSN=amqp://localhost/%2f/messages?auto_setup=false
+
+.. configuration-block::
```

Comment:
Currently only Doctrine and Beanstalkd had this section. It applies to all transports to I moved it up.